workshop/mathIsFun/ComplexNumber.cpp

```diff
@@ -0,0 +1,165 @@
+#include "ComplexNumber.h"
+#include <string>
+
+/*
+ * default constructor, initialize number to be zero
+*/
+ComplexNumber::ComplexNumber() :
+    m_real(0), 
+    m_imaginary(0)
+{}
```

Comment:
I recommend adding a comment explaining this function was left blank intentionally.